### PR TITLE
Completed html-forms | Rocky Truong

### PR DESCRIPTION
**Pull request by rocky-truong**
image

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      <div>
+        <select id="reason" name="reason">
+          <option value="default">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+    </label>
+    <label for="languages">
```
Comment (codingcodymiller) [inline html-forms/index.html:26]: This doesn't need to be a label, there should be a `label` element around each option's text

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      Favorite Language (choose one)
+      <div>
+        <input type="radio" name="language" value="html" /> HTML
+        <input type="radio" name="language" value="css" /> CSS
+        <input type="radio" name="language" value="javascript" /> JavaScript
+      </div>
+    </label>
+    <label for="information">
```
Comment (codingcodymiller) [inline html-forms/index.html:34]: This shouldn't be a label, the label should be around "Sign me up!"

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    <label for="comments">
+      Comments
+      <div>
+        <textarea name="comments" cols="25" rows="10">
+        </textarea>
+      </div>
+      <label for="submit">
+        <input type="submit" name="submit" value="Submit" />
+      </label>
+    </label>
```
Comment (codingcodymiller) [inline html-forms/index.html:49]: The labels in this section don't have an `id` on their correlated input element that matches the `for` attribute value

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
A few fixes for you!

**Comment by rocky-truong** [comment]
image

File: html-forms/index.html [hunk 4]
```diff
@@ -23,29 +23,34 @@ <h1>Coding Form!</h1>
...
-    </label>
-    <label for="information">
+    </span>
+    <div>
+      <input type="radio" name="language" value="html" />
+      <label for="HTML">HTML</label>
+      <input type="radio" name="language" value="css" />
+      <label for="CSS">CSS</label>
+      <input type="radio" name="language" value="javascript" />
+      <label for="JavaScript">JavaScript</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:35]: These `input` elements need an `id` attribute that matches the `for` attribute on the label

File: html-forms/index.html [hunk 5]
```diff
@@ -23,29 +23,34 @@ <h1>Coding Form!</h1>
...
+    <span>
       Would you like to sign up for more information?
-      <div>
-        <input type="checkbox" name="signup" /> Sign me up!
-      </div>
-    </label>
+    </span>
+    <div>
+      <label for="signup">
+        <input type="checkbox" id ="signup" name="signup" /> Sign me up!
```
Comment (codingcodymiller) [inline html-forms/index.html:42]: Please ensure there are no spaces around the equal sign in attributes, it can cause parsing errors in the browser

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Couple more minor changes!

**Comment by rocky-truong** [comment]
image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**